tests/utils/storageclass.go

```diff
@@ -57,3 +59,24 @@ func DeleteStorageClass(clientSet *kubernetes.Clientset, storageClass *storageV1
 		return false, err
 	})
 }
+
+// GetStorageClass uses the provided client to attempt to get the specified storage class by name
+func GetStorageClass(clientSet *kubernetes.Clientset, scName string) (*storageV1.StorageClass, error) {
+	sc, err := clientSet.StorageV1().StorageClasses().Get(scName, metav1.GetOptions{})
+	if err != nil {
+		fmt.Printf("\nOk... I expected this error so wth: %v\n", err)
```

Comment:
I would do an errors.Wrap here, instead of a Printf that may or may not end up somewhere.

Reply:
Oops, that's actually a debug line I had in there; never meant for you to  see it :)